### DISCUSSION TITLE
Help with Starting in Haxe

### DISCUSSION THREAD

**Discussion by bwkam** [HaxeFlixel/flixel#2419] [Q&A] [answered]
Hi there,
I'm a web developer, and I'm looking forward to get started with making games in Haxe with HaxeFlixel like friday night funkin',
but sadly, I don't find too much resources and I can't find any course, so please could someone help me with that, and share
with me how you started and became good in the language?

**Comment by RichardBray** [comment, marked as answer]
I think there are loads of resources out there. 
- The first place I'd recommend checking out is this [Awesome Haxeflixel repo](https://github.com/And-0/awesome-haxeflixel) which should have enough links to resources to help you get started
- If you haven't done so already I'd recommend joining out the [HaxeFlixel Discord channel](https://discord.gg/QrCqtFKY) to ask more specific questions if you have them
- There is a great [starter tutorial on the Haxeflixel website](https://haxeflixel.com/documentation/tutorial/)

**Reply by Geokureli** [reply]
^ All of this, 100%. But it also helps to check out the [Demos page](https://haxeflixel.com/demos/) for a comprehensive list of features and how-tos. Every demo has a link to it's source.

I learned game programming by trying to reproduce some of my favorite game's mechanics.

**Reply by bwkam** [reply]
Thanks too much for these resources, I will follow them
